mw-webapp/src/component/header/Header.module.scss

```diff
@@ -11,7 +11,15 @@
 
 .title {
   display: flex;
-  align-items: center;
+  align-items: flex-start;
+  justify-content: center;
+  color: $button-active;
+  text-shadow: 2px 2px 2px $button-shadow;
+  margin-top: -1em;
```

Comment:
Is it right that title hase button styles?

Reply:
changed

Reply:
It's not good idea to use for color rgd(...). Better to use global var with color )